SUMMARY.md

```diff
@@ -122,6 +122,8 @@
 		* [POST Transcribe recording](voice/methods/recordings/postCallsCallIdRecordingsRecordingIdTranscription.md)
 		* [GET Download Transcription](voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md)
 		* [DELETE Delete Transcription](voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md)
+	* [Conferences](voice/methods/conferences/about.md)
+	    * [PUT Update Conference Member](voice/methods/conferences/putConferenceMember.md)
```

Comment:
The other files are named after their uri. So to follow the same pattern this should be named `putConferencesConferenceIdMembersMemberId.md`.